mbed_build/_internal/config/config_modifiers.py

```diff
@@ -59,6 +61,6 @@ def build_modifier_from_config_entry(key: str, data: Any) -> Callable:
 
 
 def build_modifier_from_target_override_entry(key: str, data: Any) -> Callable:
-    """Target overrides come in three shapes: addition, removal and override."""
+    """TODO."""
```

Comment:
Do you have another PR lined up to address this `TODO`? If not could we have some more context in the comments as to what needs to be done? (In case I have to pick this up later).

Reply:
It'll be addressed with cumulative overrides PR, but I should definitely move the `TODO` from the line below. 👍

Reply:
https://github.com/ARMmbed/mbed-build/pull/19/commits/cde6268ff6e6be3058b184f456fd15d04925cded